mvc/src/main/java/com/interface21/webmvc/servlet/view/InternalResourceViewResolver.java

```diff
@@ -0,0 +1,11 @@
+package com.interface21.webmvc.servlet.view;
+
+public class InternalResourceViewResolver implements ViewResolver {
+
+    private static final String DEFAULT_EXTENSION = ".jsp";
+
+    @Override
+    public View resolveViewName(String viewName) {
+        return new JspView(viewName + DEFAULT_EXTENSION);
+    }
+}
```

Comment:
```suggestion
public class InternalResourceViewResolver implements ViewResolver {
    private static final String DEFAULT_EXTENSION = ".jsp";
    
    private final String extension;

    public InternalResourceViewResolver() {
        this(DEFAULT_EXTENSION);
    }

    public InternalResourceViewResolver(String extension) {
        this.extension = extension;
    }

    @Override
    public View resolveViewName(String viewName) {
        return new JspView(viewName + extension);
    }
}
```

이전 코드에서 생성자를 통해 확장성을 고려해봐도 좋겠네요~! 😀